### ISSUE TITLE
[wg/pointer-events] Pointer Events Working Group rechartering

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#515]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/charter-drafts/2025/pointer-events-wg.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/charter-drafts/2025/pointer-events-wg.html)

Expected end of charter refinement phase: 2025-12-04

If applicable:

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https://www.w3.org/2023/11/wg-pointer-events-charter.html&doc2=https://w3c.github.io/charter-drafts/2025/pointer-events-wg.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/pointer-events)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 

## Communities suggested for outreach

None. ~This is pure maintenance.~

This charter adds gesture within its scope and make it clearer that we're taking over mouse events.

## Known or potential areas of concern

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

Raise [issues](https://github.com/w3c/charter-drafts/issues?q=is%3Aissue%20state%3Aopen%20in%3Atitle%20pointer-events)

## Anything else we should think about as we review? 

Nope

cc @patrickhlauke

**Comment by simoneonofri** [comment]
no comment or request from SING.

**Comment by patrickhlauke** [comment]
@plehegar should i go and make edits based on the TAG feedback to the proposed charter now, or do we need to wait for any others to complete first?

EDIT: never mind, beat me to it. thank you.

**Comment by dontcallmedom** [comment]
a few comments:
* if PE Level 4 is where gestures are expected to be defined, this should probably be made explicit in the description of the deliverable
* especially given the history in this space, I think it would be much preferable if there was an incubated spec that described these high-level gestures, or at least a list of which gestures would be in scope
* the first bullet of "out of scope" remains confusing (it starts with "internals", when then goes to talk about "gesture functionality and APIs"); rather than optimizing for minimal changes to the charter, I think this should be rewritten for maximal clarity
* the Touch Community Group was closed in 2024, so shouldn't be listed in the coordination section

**Comment by pchampin** [comment]
typo in Section 2 Scope: "only have use a single model". It seems that either "have" or "use" should be removed.

**Comment by daniel-montalvo** [comment]
See [typo on the name of the Web Applications Working Group](https://github.com/w3c/charter-drafts/pull/741).

**Comment by plehegar** [comment]
We need to draft use cases and requirements. See https://www.w3.org/2026/01/28-pointerevents-minutes.html#2a49

This means that our Charter refinement period will extend until April while we're working on it.

**Comment by plehegar** [comment]
We punted to the CG for now to provide a proposal for gesture events. We [removed gesture events](https://github.com/w3c/charter-drafts/pull/803) from the charter in the meantime.

See [May 6, 2026 minutes](https://www.w3.org/2026/05/06-pointerevents-minutes.html#d556)

**Comment by plehegar** [comment]
https://github.com/w3c/transitions/issues/779 got approved but waiting on the short extension to trigger the AC review.

**Comment by pchampin** [comment]
Minor remarks:

* Q2 2026 for 'Pointer Events (Level 3)' seems overly optimistic by now
* copyright should probably be 2026 instead of 2025

**Comment by plehegar** [comment]
> Minor remarks:
> 
> * Q2 2026 for 'Pointer Events (Level 3)' seems overly optimistic by now

We got approval to start the AC review for that document but we're waiting on the start of the charter review to trigger it...

> * copyright should probably be 2026 instead of 2025

Fixed by https://github.com/w3c/charter-drafts/commit/27b5b149885e8fad5360add77333c7d11db10e6a

**Comment by plehegar** [comment]
[VOTE by 2026-06-22: Proposed Charter for the Pointer Events Working Group](https://lists.w3.org/Archives/Member/w3c-ac-members/2026AprJun/0043.html)

**Comment by plehegar** [comment]
> [w3c/transitions#779](https://github.com/w3c/transitions/issues/779) got approved but waiting on the short extension to trigger the AC review.

[VOTE by 2026-06-24: Proposed Advancement of Pointer Events Level 3 to W3C Recommendation (Call for Review)](https://lists.w3.org/Archives/Member/w3c-ac-members/2026AprJun/0048.html)

**Comment by plehegar** [comment]
[Call for Participation: Pointer Events Working Group Charter approved; Join the Pointer Events Working Group](https://lists.w3.org/Archives/Public/public-new-work/2026Jun/0021.html)
